README.adoc

```diff
@@ -150,6 +150,10 @@ parameters:
    rhn_username: ""
    rhn_password: ""
    rhn_pool: '' # OPTIONAL
+   
+   # Currently Ansible 2.1 is not supported so add these parameters as a workaround
+   openshift_ansible_git_url: https://github.com/openshift/openshift-ansible.git
+   openshift_ansible_git_rev: 08791978fdf3ee385760761d4fc6bc47febf1732
```

Comment:
Could you please update https://github.com/redhat-openstack/openshift-on-openstack/blob/master/env_origin.yaml#L19 ? Otherwise LGTM.